api/src/main/java/au/org/aodn/nrmn/restapi/validation/process/RawValidation.java

```diff
@@ -145,13 +146,13 @@ public StagedRowFormatted toFormat(HashMap<String, Object> values, boolean isExt
         val method = (Integer) values.get("Method").orElseGet(null);
         val block = (Integer) values.get("Block").orElseGet(null);
 
-        val species = Optional.ofNullable((ObservableItem) values.get("Species").orElseGet(null));
-        //val speciesAttributesOtp = obsRepo.getSpeciesAttributesById(new Long(species.getObservableItemId()));
-//        val speciesAttributes = speciesAttributesOtp
-//                .stream()
-//                .findFirst()
+        val species = (ObservableItem) values.get("Species").orElseGet(null);
+        val speciesAttributesOtp = obsRepo.getSpeciesAttributesById(new Long(species.getObservableItemId()));
```

Comment:
Species can be null for "Survey not done" so this can result in a null pointer exception.

Reply:
Optional<StagedRowFormatted>  is a much better way of handling that